src/components/ZetkinSpeedDial/actions/createTask.tsx

```diff
@@ -20,7 +20,7 @@ const DialogContent: React.FunctionComponent<DialogContentBaseProps> = ({ closeD
         closeDialog();
         const newTask = await eventMutation.mutateAsync(data);
         // Redirect to task page
-        router.push(`/organize/${orgId}/campaigns/${campId}/tasks/${newTask.id}`);
+        router.push(`/organize/${orgId}/campaigns/${campId}/calendar/tasks/${newTask.id}`);
```

Comment:
here too, do we need `closeDialog()` @richardolsson ? Here it's before we await the promise, in `createCampaign` it's after -- don't know what that's doing

Reply:
Very valid point!

Lets get rid of the `closeDialog()` and check that everything is still working. Can you try that, @omidhq?

Reply:
Good point @nehahirve. It works perfectly without that function. Made the changes and pushed it again, @richardolsson.